registry/strings/helpers.go

```diff
@@ -93,17 +93,6 @@ func (sr *StringsRegistry) initials(str string, delimiters string) string {
 //
 //	string - the transformed string.
 //
-// Example:
-//
-//	style := caseStyle{
-//	    Separator:       '_',
-//	    CapitalizeNext:  true,
-//	    ForceLowercase:  false,
-//	    InsertSeparator: true,
-//	}
-//	transformed := sr.transformString(style, "hello world")
-//	Output: "Hello_World"
```

Comment:
This could be move to an ExampleXXX test method no?

I mean the example was interesting